src/code.cloudfoundry.org/silk/controller/integration/integration_test.go

```diff
@@ -302,11 +320,21 @@ var _ = Describe("Silk Controller", func() {
 			Context("when the existing lease is in a different overlay network", func() {
 				BeforeEach(func() {
 					helpers.StopServer(session)
-					conf.Network = "10.254.0.0/16"
+					conf.Network = []string{"10.254.0.0/16"}
 					session = helpers.StartAndWaitForServer(controllerBinaryPath, conf, testClient)
 				})
+
 				It("renews the same lease in the old network", func() {
-					err := testClient.RenewSubnetLease(existingLease)
+					By("validating that the old lease is not in the new network")
```

Comment:
This test case was a little confusing to me. would it be better to reframe it as renewing the lease in the same overlay as the old lease was in, regardless of which overlay it was?